rott/rt_main.c

```diff
@@ -815,17 +815,15 @@ NoRTC:;
 
 #endif
 
-   // Normal ROTT wads
+	// Normal ROTT wads
+	filename = M_StringJoin(datadir, PATH_SEP_STR, STANDARDGAMEWAD, NULL);
+	if ((newargs[argnum++] = M_FileCaseExists(filename)) == NULL)
```

Comment:
`M_FileCaseExists()` always creates a duplicate of the passed string, so `free(filename)` needs to be called in any case.